dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -337,6 +339,30 @@ class ScriptSeedTest < ActiveSupport::TestCase
       )
     end
 
+    test 'seed updates lesson programming expressions' do
+      script = create_script_tree
+
+      script_with_changes, json = get_script_and_json_with_change_and_rollback(script) do
+        lesson = script.lessons.first
+        lesson.programming_expressions.first.update!(name: 'newBlock')
+        lesson.programming_expressions.create(
+          key: 'newBlock',
+          name: 'newBlock',
+          programming_environment_id: 1
+        )
```

Comment:
this part is tricky. you have to create the actual PE outside of the block. see this part of my pr on lessons standards: https://github.com/code-dot-org/code-dot-org/blob/eb6ca095dab6ca52b8e70c09988faf59d5f2361f/dashboard/test/lib/services/script_seed_test.rb#L345-L347